### DISCUSSION TITLE
Is there a way to filter all archived WFs in "Workflows" page?

### DISCUSSION THREAD

**Discussion by ni-todo-spot** [argoproj/argo-helm#2841] [Q&A]
Hey all!

Just upgraded our argo to the newest version and noticed there is no more archive page, all WFs are in one page with 1 field indicating whether it's cached or not.
This got me to think if can filter all archived WFs and view those which are not.

Thanks in advance!

**Comment by yu-croco** [comment]
Hi @ni-todo-spot , since it's about specific feature of Argo Workflows, please ask at [upstream](https://github.com/argoproj/argo-workflows).

**Reply by ni-todo-spot** [reply]
@yu-croco my apologies - I did not intended to post it here sorry
